examples/using-gatsby-without-graphql/README.md

```diff
@@ -4,7 +4,7 @@ Many examples in the Gatsby docs focus on using source plugins to pull data into
 
 This example loads data from the [PokéAPI](https://www.pokeapi.co/)’s REST endpoints, then creates pages (and nested pages) using [Gatsby’s `createPages` API](https://www.gatsbyjs.org/docs/node-apis/#createPages).
 
-You might also be interested in this [blog post on unstructured data](/blog/2018-10-25-unstructured-data/), or the relevant [docs page](/docs/using-unstructured-data/).
+You might also be interested in this [blog post on unstructured data](https://www.gatsbyjs.org/blog/2018-10-25-using-gatsby-without-graphql/#an-example-of-creating-pages-using-unstructured-data-from-a-remote-api), or the relevant [docs page](https://github.com/gatsbyjs/gatsby/blob/master/docs/docs/using-gatsby-without-graphql.md).
```

Comment:
Let's stick to relative links

Reply:
This example README is only read on GitHub, not on our site 🤔 
So the second changed link needs to go to our website as an absolute link

Reply:
Hi @sidharthachatterjee @LekoArts , thank you for the suggestions. I changed the "blog post" link to be a relative link and the "docs page" link to go to the website. Please let me know if there are any other changes you'd recommend!